### DISCUSSION TITLE
Hello coders and coderettes

### DISCUSSION THREAD

**Discussion by AndreHaupt85** [neuefische/questions#259] [Web] [answered]
Hey hey :)

my gallery is moving sideways on his own with a function.

![Bildschirmfoto 2022-10-13 um 10 36 41](https://user-images.githubusercontent.com/109967968/195548120-efa7a18b-d463-43d1-9edb-b39514f76e34.png)

i have 2 "problems" right now.

first of all i would like to remove the horizontal scrollbar. i tryed a couple of solutions of google but non of them was working.
![Bildschirmfoto 2022-10-13 um 10 35 42](https://user-images.githubusercontent.com/109967968/195548235-88eddb93-efe0-4c34-b901-7ee57dad494f.png)
![Bildschirmfoto 2022-10-13 um 10 35 27](https://user-images.githubusercontent.com/109967968/195548243-d751a975-1101-4fb3-9007-1c416e14c1d4.png)
![Bildschirmfoto 2022-10-13 um 10 35 21](https://user-images.githubusercontent.com/109967968/195548259-b8b07bcc-a8a8-402b-8687-df62cbbaee52.png)

and second. its still possible to scroll sideways with the mouse wheel and i would prefer that its only moving with the function. but impossile to move by mouse.

Maybe someone has a solution

**Comment by ThomasKirst** [comment, marked as answer]
Hi Andre,
you are quite close to the solution. It should be sufficient to add `overflow: hidden` to your `Gallery` component:

```js
const GalleryBox = styled.div`
  max-width: 280px;
  height: 355px;
  display: flex;
  overflow: hidden;
  border-radius: 10px;
  box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px,
    rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px,
    rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
`;
```

EDIT: Once the scrollbars are gone, the problem with scrolling by the mouse wheel should be gone, right?

–Thomas
